inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -59,6 +59,8 @@
 
 #if GNA_LIB_VER == 2
 #include <gna2-model-api.h>
+//#include <transformations/common_optimizations/swap_input_matmul_gna.hpp>
+#include <ngraph_transformation/include/swap_input_matmul_gna.hpp>
```

Comment:
We already have a folder 'transformations' for ngraph transformations.
I can't find this file (ngraph_transformation/include/swap_input_matmul_gna.hpp) in this pull request.